cmd/src/cli/run.rs

```diff
@@ -0,0 +1,47 @@
+use cli::{self, package};
+use error::DefaultResult;
+use holochain_container_api::{
+    config::{AgentConfiguration, Configuration, DNAConfiguration},
+    container::Container,
+};
+
+pub fn run(package: bool) -> DefaultResult<()> {
+    if package {
+        cli::package(true, Some(package::DEFAULT_BUNDLE_FILE_NAME.into()))?;
+    }
+
+    let agent_config = AgentConfiguration {
+        id: "hc-run".into(),
+        key_file: "hc_run.key".into(),
+    };
+
+    let dna_config = DNAConfiguration {
+        id: "dna.local".into(),
+        file: package::DEFAULT_BUNDLE_FILE_NAME.into(),
+        hash: "Qm328wyq38924ybogus".into(),
+    };
+
+    let base_config = Configuration {
+        agents: vec![agent_config],
+        dnas: vec![dna_config],
+        ..Default::default()
+    };
+
```

Comment:
We need an `InstanceConfiguration` and an `InterfaceConfiguration`, no?

Reply:
Does this work without??

Reply:
🤷‍♂️ 
Give me some pointers. It's not really easy to create these configurations

Reply:
https://github.com/holochain/holochain-rust/blob/develop/container/example-config/basic.toml